### ISSUE TITLE
bug(@vtmn/svelte): problem when importing `@vtmn/svelte` in our Svelte stack 

### ISSUE THREAD

**Issue by lauthieb** [Decathlon/vitamin-web#648]
# Bug Report

Hello, after trying to import `@vtmn/svelte` inside our Svelte stack for the e-commerce with @jleveugle & @xavhan, it doesn’t work because our Svelte library components import the styles of each component via their specific npm packages _(example for VtmnButton, the import of `@vtmn/css-button`)_.

The e-commerce Svelte stack of Decathlon does not have this dependency, so it breaks at compile time. So we have to find a solution to make the stack work. Here are two proposed solutions:
1. Provide .js, .mjs and .svelte output by injecting the style at each component level. This would allow installing the `@vtmn/svelte` library only and to directly benefit from the style of each component. It will also be necessary to propose a file of type index allowing to gather all the export in the same place to facilitate the consumption. The problem concerning our Svelte stack will be that the injected styles will be in base 16 (e.g: 1rem = 16px) and that if we import `@vtmn/css` in parallel with `@vtmn/svelte` components, there would be duplicate CSS _(example: twice the presence of the button style, etc.)_
2/ Do not provide the style and put `@vtmn/css` in dependencies or peerDependencies of the library `@vtmn/svelte`.

Which solution would be better according to you?

I suggest that we discuss it here and of course we are open to discussion in videoconference if it can help!

The goal is that the `@vtmn/svelte` package can be used more and more in order to abstract from the Vitamin logic and to use as little as possible the CSS classes of components.

Thanks in advance for your opinions! :)

**Comment by lauthieb** [comment]
@jleveugle @thollander @mlefebvre-decathlon @Margot-ElMadi @MaxenceDelaine2 @xavhan @pdelap26
It would be nice to have your opinion 🙏 ❤️

**Comment by lauthieb** [comment]
Fixed! Thanks @anthonylegoas for the great work on this 💪
